### ISSUE TITLE
3d directory

### ISSUE THREAD

**Issue by yangli0516** [HughCraig/TLCMap#61]
The current '3d' directory under the public folder contains 'place.html' and 'places.html' in use, while other files are not used anywhere.
Could completely remove this directory and move those two place files into 'view' directory. Links need to be changed also.

**Comment by BillPascoe** [comment]
Yes clean up those folders and move but set up a redirect, maybe an
htaccess file,  for those who have used the old link.
